src/content/pt-br/fundamentals/security/encrypt-in-transit/why-https.md

```diff
@@ -0,0 +1,70 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: You should always protect all of your websites with HTTPS, even if they don’t handle sensitive communications. HTTPS provides critical security and data integrity both for your websites and for the people that entrust your websites with their personal information.
+
+{# wf_updated_on: 2016-08-22 #}
+{# wf_published_on: 2015-11-23 #}
+
+# Por que HTTPS é importante {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+<div class="video-wrapper">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="9WuP4KcDBpI"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+Você deve sempre proteger todos os seus sites com HTTPS, mesmo que eles não
+trabalhem com comunicações sensíveis. Além de oferecer segurança crítica e integridade dos dados
+do seus sites e das informações pessoais de seus usuários, o HTTPS
+Um requisito para muitos recursos dos navegador mais recentes, particularmente aqueles necessários 
+em [aplicativos web progressivos](/web/progressive-web-apps/).
+
+### TL;DR {: .hide-from-toc }
+
+* Tanto intrusos malignos quanto benignos exploram todos os recursos desprotegidos entre seus sites e os usuários.
+* Muitos intrusos analisam uma série de comportamentos para identificar seus usuários.
+* O HTTPS não bloqueia apenas o mau uso do seu site. Ele Também é um requisito para muitos recursos de ponta e uma tecnologia habilitadora para recursos semelhantes a aplicativos, como os manipuladores de serviços.
```

Comment:
I'm not sure if "service worker" should be translated. Is it translated elsewhere? It sounds weird.